index.html

```diff
@@ -12,342 +12,347 @@
 </head>
 <body>
   <div id="blurBg">
-    <header>
-      <a href="#"><h2>Owusu Dennis</h2></a>
-      <div class="menuImg"><img src="./images/menu.png" class="menu" alt="Menu"></div>
+    <header class="main-header">
+      <a class="logo-name" href="#"><h2 class="owusu">Owusu Dennis</h2></a>
 
       <nav>
-        <ul class="menu_items" id="menuList">
+        <button class="menuImg"><img src="./images/menu.png" alt="Menu"></button>
+        <ul class="menu_items" id="menu_items">
```

Comment:
[ ] the close button is missing from your hamburger button. kindly add it to match the Figma design.
    -When the user clicks (or taps) the close (X) button, the mobile menu disappears.   
Your design :
![hamte3oi](https://user-images.githubusercontent.com/62156607/208757367-9825de75-b069-4e10-a14c-c6f7d95572ed.PNG)

Figma design : 
![hamfigma](https://user-images.githubusercontent.com/62156607/208757391-b9f6e22f-770c-4af4-800f-cd6c69dc2c72.PNG)

Reply:
Alright